data/account/participation.go

```diff
@@ -48,7 +48,7 @@ type Participation struct {
 
 	VRF    *crypto.VRFSecrets
 	Voting *crypto.OneTimeSignatureSecrets
-	// StateProofSecrets is used to sign compact certificates. might be nil
+	// StateProofSecrets is used to sign compact certificates. MIGHT BE NIL!
```

Comment:
nothing in this line is wrong; but anytime we have a pointer variable, we should assume that it could be nil, unless we explicitly verify that.